server/server.js

```diff
@@ -315,7 +410,44 @@ app.get("/countries/:country", async function (req, res) {
   }
   res.send(country);
 });
+app.get("/countries/:country/all", async function (req, res) {
+  if (req.params.country !== "usa") {
+    res.status(200).json([]);
+    return;
+  }
 
+  let states = JSON.parse(await redis.get(keys.usa_states))
+  if (req.query['sort']) {
+    try {
+      const sortProp = req.query['sort'];
+      states.sort((a, b) => {
+        if (a[sortProp] < b[sortProp]) {
+          return -1;
+        } else if (a[sortProp] > b[sortProp]) {
+          return 1;
+        }
+        return 0;
+      })
+    } catch (e) {
+      console.error("ERROR while sorting", e);
+      res.status(422).send(e);
+      return;
+    }
+  }
+  res.send(states);
+});
+// app.get("/countries/usa/:state", async function(req, res) {
+//   const stateName = req.params.state;
+//   let usaStates = JSON.parse(await redis.get(keys.usa_states))
```

Comment:
Delete commented out code.

Reply:
I didn't know if you would find it useful to get individual states. If you dont think there is use for it, i'll remove it.